cmd/cmdutil/helpers.go

```diff
@@ -47,6 +47,16 @@ func AddLimitFlag(cmd *cobra.Command, limit int, desc string) {
 	cmd.PersistentFlags().IntP("limit", "l", limit, desc)
 }
 
+// AddMaxConcurrencyFlag adds --max-concurrency to command
+func AddMaxConcurrencyFlag(cmd *cobra.Command, max int, desc string) {
+	cmd.PersistentFlags().IntP("max-concurrency", "m", max, desc)
```

Comment:
I think these flags should be strings; [per AWS docs](https://docs.aws.amazon.com/sdk-for-go/api/service/ssm/#SendCommandInput):

>You can specify a number like 10 or a percentage like 10%.

We might also want to add some validation that the values are either a number or a numeric string ending in "%". :)

Reply:
Also, the default value for MaxConcurrency is "50". MaxErrors is still "0".